magento/magento2
# fix incorrect image size when rendition is inserted

### Description (*)

Fixes issue that image uploader displays incorrect image size when rendition is inserted from media gallery
### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1806

### Manual testing scenarios (*)

#### Preconditions:
1. Use old media gallery - No
2. Media Gallery Renditions Configured
#### Steps to test:
1. Open category page
2. Open content section
3. Insert Image from media gallery
4. Insert image with high resolution and file size
5. Ensure that image resolution and size responds to rendition image
